### ISSUE TITLE
New CruX metrics

### ISSUE THREAD

**Issue by samunderwood** [edgemesh/crux.run#10]
Hey,

Are there plans to update this to support new metrics in CruX such as cumulative layout shift and largest contentful paint?

**Comment by jacobloveless** [comment]
Yes we're looking into adding First Input Delay , CLS and LCP in a (near term) release.

We're working with the Google team on the exact key metrics to bring in (and they'll be in the API first then into the UI shortly after).

Thanks for the question!

Sent via Superhuman ( https://sprh.mn/?vip=jacob.loveless@edgemesh.com )

**Comment by samunderwood** [comment]
Awesome! Another question, in the UI how are you categorising FCP as fast, average and slow? Strugglign to find any pre-defined metrics for this categorisation? It doesn’t look like you surface this in the API.

**Comment by jacobloveless** [comment]
You know … we sure don't!

So a:

fast first paint is <1000ms, and a slow is >2500ms

fast first contentful paint is <1000ms, and a slow is >2500ms

fast content load is <1500ms, and a slow is >3500ms

These are all modeled after PageSpeed insights. Will get that added as a route!

Sent via Superhuman ( https://sprh.mn/?vip=jacob.loveless@edgemesh.com )

**Comment by samunderwood** [comment]
Awesome, thanks!

**Comment by samunderwood** [comment]
Is there any ETA for when you think you'll be able to add the new CrUX metrics to the API? 😊

**Comment by jacobloveless** [comment]
Not the next release (May) but the one after (June).

Sent via Superhuman iOS ( https://sprh.mn/?vip=jacob.loveless@edgemesh.com )

**Comment by samunderwood** [comment]
This still planned for release this month? 😊

**Comment by jacobloveless** [comment]
working on the rewrite of the parser as we speak!

**Comment by samunderwood** [comment]
Awesome, I have a blog post prepped for when this goes live to show how to use it with Google Sheets 😊
